doc/source/index.rst

```diff
@@ -0,0 +1,73 @@
+.. PBxplore API documentation master file, created by
+   sphinx-quickstart on Tue Oct  6 15:44:18 2015.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to PBxplore's documentation!
+====================================
+
+**PBxplore is a suite of tools dedicated to Protein Block analysis.**
+
+Protein Blocks (PBs) are structural prototypes defined by
+`de Brevern <http://www.dsimb.inserm.fr/~debrevern/index.php>`_ *et al* in 2000 [#]_.
+The 3-dimensional local structure of a protein backbone can be modelized as an 1-dimensional
+sequence of PBs.
+In principle, any conformation of any amino acid could be represented
+by one of the sixteen available Protein Blocks.
+
+.. figure:: img/PBs.jpg
+    :align: center
+
+    Schematic representation of the sixteen protein blocks, labeled from *a* to *p*
+    (Creative commons CC-BY).
```

Comment:
A link to the license would be useful. Also it would tell what license exactly we are talking about.